Formula/elasticsearch@6.rb

```diff
@@ -43,6 +42,9 @@ def install
       s.sub!(%r{#\s*path\.logs: /path/to.+$}, "path.logs: #{var}/log/elasticsearch/")
     end
 
+    inreplace "#{libexec}/config/jvm.options", %r{logs/gc.log}, "#{var}/log/elasticsearch/gc.log"
+    inreplace "#{libexec}/config/jvm.options", /10-:-XX:UseAVX=2/, "# 10-:-XX:UseAVX=2"
```

Comment:
```suggestion
    inreplace "#{libexec}/config/jvm.options" do |s|
      s.gsub! "logs/gc.log", "#{var}/log/elasticsearch/gc.log"
      s.gsub! "10-:-XX:UseAVX=2", "# 10-:-XX:UseAVX=2"
    end
```
Should commenting out the AVX line only be done on ARM?